docs/nodes/validator-node-handling.md

```diff
@@ -473,11 +476,18 @@ If you're looking for how to install a statesynced node follow the instructions
 
 In this guide, we will walk you through the process of reducing the disk usage of your Commercio Network node by state syncing it to the blockchain. 
 
-:::warning
-Please note that the following commands assume you are acting as the root user. Adjust the user and file paths accordingly if your node is using a different user.
-:::
+1. Set the chain variables to adapt the procedure to any chain user (root or cnd)
```

Comment:
**Warning**: You can't be sure of the configuration that node managers have: either you explicitly write that a configuration of a certain type is expected or you risk them doing damage. We can discuss this better eventually